piconumpy/_piconumpy_hpy.c

```diff
@@ -1,23 +1,24 @@
 #include "hpy.h"
 
 typedef struct {
-  HPyObject_HEAD
   /* Type-specific fields go here. */
   double *data;
   int size;
 } ArrayObject;
 
+HPyType_HELPERS(ArrayObject)
```

Comment:
I don't get why these diffs are here because they are already in the `hpy` branch: https://github.com/hpyproject/piconumpy/blob/hpy/piconumpy/_piconumpy_hpy.c